composer.json

```diff
@@ -122,6 +122,6 @@
         "phing-install-dependencies": ["phing patch-dependencies", "phing installsolr installswaggerui"],
         "post-install-cmd": "@phing-install-dependencies",
         "post-update-cmd": "@phing-install-dependencies",
-        "qa": "phing qa-console"
+        "qa": "phing qa-console -Dlocaldir=$(pwd)/qa_local"
```

Comment:
A few questions:

- I'm having trouble finding documentation about `$(pwd)` -- is this reliable cross-platform, or if you run composer outside of the working directory (which is possible with the `--working-dir` switch)? I can test if you like; just curious if you've already done that investigation.
- Is this going to leave behind a qa_local directory with junk in it? Do we want to clean up after ourselves, or put this in the /tmp directory (if there's a platform-agnostic way to do that), or at least .gitignore it?

Reply:
Now I'm not sure. I just found it in some examples, but you're right, it might not work correctly for all cases. 

Yes, it's going to leave junk just like before (e.g. an extra ini file). Maybe using tmp and cleaning up would work. I'll see what I can do.

Reply:
Okay, looks like there's not much that can be done in composer.json, so I resorted to adding support for a new defaultconfigs=true property that sets localdir appropriately. I also fixed the test that left a config file behind, so now all that results is an empty directory under tmp.